client/signup/steps/user/index.jsx

```diff
@@ -458,7 +458,22 @@ export class UserStep extends Component {
 	}
 
 	getHeaderText() {
-		const { flowName, oauth2Client, translate, headerText, wccomFrom, isSocialFirst } = this.props;
+		const {
+			flowName,
+			oauth2Client,
+			translate,
+			headerText,
+			wccomFrom,
+			isSocialFirst,
+			userLoggedIn,
+			locale,
+		} = this.props;
+
+		if ( userLoggedIn ) {
+			if ( englishLocales.includes( locale ) ) {
```

Comment:
Purely out of interest for myself, would you mind explaining the logic here around only showing this for English locales? Is this maybe missing a `hasTranslation` or am I misunderstanding it? :)

Reply:
You're totally right! I tested `hasTranslation` a little and couldn't get it to work properly, it was returning false for already existing translations 🤔  So I went on and thought about simply doing like this and then removing the `englishLocales` check once translations were ready!

Reply:
Ah, got it! Rather than re-visiting this (and potentially forgetting...), one other option could be to just add a String Freeze and wait for the translations here.

Although, as I think about this more, I wonder if it might make sense to use the strings from `/log-in` instead? Consistency and no translation needed 😁 Obviously your call though!

<img width="633" alt="Screenshot 2024-01-25 at 19 43 10" src="https://github.com/Automattic/wp-calypso/assets/43215253/d5af7d00-dc13-4999-b363-5b76e8d710d8">